src/Humanizer.Tests.Shared/Localisation/fr/NumberToWordsTests.cs

```diff
@@ -59,6 +59,20 @@ public void ToWords(int number, string expected)
         }
 
         [Theory]
+        [InlineData(1, "une", GrammaticalGender.Feminine)]
+        [InlineData(1, "un", GrammaticalGender.Masculine)]
+        [InlineData(2, "deux", GrammaticalGender.Feminine)]
+        [InlineData(2, "deux", GrammaticalGender.Masculine)]
+        [InlineData(121, "cent vingt et une", GrammaticalGender.Feminine)]
+        [InlineData(121, "cent vingt et un", GrammaticalGender.Masculine)]
+        [InlineData(10121, "dix mille cent vingt et une", GrammaticalGender.Feminine)]
+        [InlineData(10121, "dix mille cent vingt et un", GrammaticalGender.Masculine)]
+        public void ToWordsWithGender(int number, string expected, GrammaticalGender gender)
+        {
+            Assert.Equal(expected, number.ToWords(gender));
+        }
+
+        [Theory]
         [InlineData(0, "zéroième")]
```

Comment:
[zéroième](https://fr.wiktionary.org/wiki/z%C3%A9roi%C3%A8me) is in fact fine, but rare. :)